modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/info/item/provider/DDMStructureInfoItemFieldSetProviderImpl.java

```diff
@@ -99,7 +99,10 @@ public InfoFieldSet getInfoItemFieldSet(
 		DDMFormFieldTypeConstants.DATE, DDMFormFieldTypeConstants.NUMERIC,
 		DDMFormFieldTypeConstants.IMAGE, DDMFormFieldTypeConstants.TEXT,
 		DDMFormFieldTypeConstants.RADIO, DDMFormFieldTypeConstants.RICH_TEXT,
-		DDMFormFieldTypeConstants.SELECT
+		DDMFormFieldTypeConstants.SELECT, DDMFormFieldTypeConstants.GRID,
+		DDMFormFieldTypeConstants.DOCUMENT_LIBRARY,
+		DDMFormFieldTypeConstants.COLOR, DDMFormFieldTypeConstants.GEOLOCATION,
+		DDMFormFieldTypeConstants.JOURNAL_ARTICLE
```

Comment:
@wanderlast Have you confirmed with the product team if we can add these fields? If not, could you open a PTR for it? Thanks!